extensions/ql-vscode/src/remote-queries/run-remote-query.ts

```diff
@@ -273,6 +275,9 @@ export async function runRemoteQuery(
 
       const processedVariantAnalysis = processVariantAnalysis(variantAnalysisSubmission, variantAnalysisResponse);
 
+      const variantAnalysisManager = new VariantAnalysisManager(ctx, logger);
```

Comment:
I think we should try to re-use the variant analysis manager that is created when the extension is created. This would fire the event, but no listeners can actually be attached to it. Passing in the `variantAnalysisManager` instead of the `ctx` should be enough.

Right now that would involve passing the `VariantAnalysisManager` to the `RemoteQueriesManager`, but I think it's fine for now. We should refactor/remove the `RemoteQueriesManager` in the future to remove this dependency anyway.

Reply:
Thanks @koesie10 ! I've made the change for this. 🙇‍♀️